tests/clone3-report-ns-id.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Check appending auxstr of clone3(2) when --namespace=new is given
```

Comment:
A dot at the end of the sentence is missing here.

Reply:
fixed locally